brion/plugin/spikeReportASCII.cpp

```diff
@@ -46,8 +46,7 @@ Spikes SpikeReportASCII::read( const float )
     Spikes spikes;
     auto start = _lastReadPosition;
     _lastReadPosition = _spikes.end();
-    _currentTime = _spikes.rbegin()->first +
-                   std::numeric_limits< float >::epsilon();
+    _currentTime = UNDEFINED_TIMESTAMP;
```

Comment:
Argh, this incremental reviewing approach doesn't seem to be a good idea...

I think you should make separate PRs for independent things, for example the command line argument parsing could have gone in a separate one.